SimPPS/PPSPixelDigiProducer/data/PixelChargeMap.txt

```diff
@@ -0,0 +1,600 @@
+-75 -50 0.375935
```

Comment:
auto-generated (text) files should normally be moved to an external data area

Reply:
How can we solve this? In other similar situation, a change was need from the release management side.